tests/chaos/oadp/conftest.py

```
@@ -79,3 +80,80 @@ def drain_vm_source_node(rhel_vm_with_dv_running, oadp_backup_in_progress):
+        )
+        backup.__enter__()
+        created_backups.append(backup)
+        backup.wait_for_status(status=backup.Backup.Status.INPROGRESS, timeout=TIMEOUT_3MIN)
+        return backup
+
+    yield _start_backup
+
+    # Cleanup all created backups
+    for backup in created_backups:
+        try:
+            backup.__exit__(None, None, None)  # noqa: FCN001
```

Ruff (0.14.10)

Unused `noqa` directive (unknown: `FCN001`)

Remove unused `noqa` directive

(RUF100)